test/cholesky/lkjcholesky.jl

```diff
@@ -135,12 +135,14 @@ using FiniteDifferences
             @test m isa Cholesky{eltype(d)}
             @test Matrix(m) ≈ I
         end
-        @test_broken partype(LKJCholesky(2, 4f0)) <: Float32
+        @test partype(LKJCholesky(2, 4f0)) <: Float32
 
         @testset "insupport" begin
             @test insupport(LKJCholesky(40, 2, 'U'), cholesky(rand(LKJ(40, 2))))
             @test insupport(LKJCholesky(40, 2), cholesky(rand(LKJ(40, 2))))
             @test !insupport(LKJCholesky(40, 2), cholesky(rand(LKJ(41, 2))))
+            @test @inferred(logpdf(LKJCholesky(40, 2), cholesky(rand(LKJ(41, 2))))) === -Inf
+            @test @inferred(logpdf(LKJCholesky(40, 2f0), cholesky(Float32.(rand(LKJ(41, 2)))))) === -Inf32
```

Comment:
Similarly, here we should also test:
```suggestion
           for (d, η) in ((2, 4), (2, 1), (3, 1)), T in (Float32, Float64)
                @test @inferred(logpdf(LKJCholesky(40, T(2)), cholesky(T.(rand(LKJ(41, 2)))))) === T(-Inf)
            end
```